doc/api/http.md

```diff
@@ -4228,6 +4228,15 @@ added:
 
 Set the maximum number of idle HTTP parsers.
 
+## `WebSocket`
+
+<!-- YAML
+added:
+  - REPLACEME
+-->
+
+A browser-compatible implementation of \[`WebSocket`]\[].
```

Comment:
That's useful information. Is there a more comprehensive guide to writing docs? Maybe this would help contributors make less incorrect decisions here.